packages/access/Scarb.toml

```diff
@@ -1,16 +1,23 @@
 [package]
 name = "openzeppelin_access"
+readme = "README.md"
+keywords = [
+    "openzeppelin",
+    "starknet",
+    "cairo",
+    "contracts",
+    "standards",
+    "access"
+]
```

Comment:
I don't think adding "cairo" or "starknet" to our packages is very valuable(as adding "rust" wouldn't be very valuable in crates.io). Also the [official docs](https://docs.swmansion.com/scarb/docs/reference/manifest.html#keywords) recommends not adding more than 5 keywords to any package